samples/subsys/usb/hid-mouse/README.rst

```diff
@@ -0,0 +1,70 @@
+.. _usb_hid-mouse:
+
+USB HID mouse Sample Application
+####################################
+
+Overview
+********
+
+This sample app demonstrates use of a USB HID driver provided by the zephyr
```

Comment:
How about:

    This sample app demonstrates use of a USB Human Interface
    Device (HID) driver by the Zephyr project.  This very simple driver
    enumerates a board with a button into a "mouse" that only has
    a left-mouse button and can't move.